docs/releases/1.11.txt

```diff
@@ -205,7 +205,9 @@ Database backend API
 Miscellaneous
 -------------
 
-* ...
+* The :meth:`~django.utils.feedgenerator.SyndicationFeed.latest_post_date` now
+  returns current UTC date/time if no items have either of the
```

Comment:
the current UTC date/time instead of a datetime without any timezone information